soc/arm/infineon_cat1/psoc6/Kconfig.soc.psoc6_02

```diff
@@ -172,6 +172,12 @@ config SOC_CY8C6248LQI_S2D02
 	select SOC_PACKAGE_PSOC6_02_68_QFN
 	depends on SOC_SERIES_PSOC_62
 
+config SOC_CY8C6244LQI_S4D92
```

Comment:
move this to soc/arm/infineon_cat1/psoc6/Kconfig.soc.psoc6_04 (need to create)

Reply:
please use Kconfig.soc.psoc6_04 and Kconfig.defconfig.soc.psoc6_04 from attached archive ([psoc6.zip](https://github.com/zephyrproject-rtos/zephyr/files/12076883/psoc6.zip)
). It has defined all NPMs for psoc6_04

Reply:
done but I have different values for `NUM_IRQS` based on [datasheet](https://www.infineon.com/dgdl/Infineon-PSoC_6_MCU_CY8C62X4-DataSheet-v12_00-EN.pdf?fileId=8ac78c8c7ddc01d7017ddd026d585901)

I see 175 for cm4 and 16 for cm0 (8 hw and 8 sw irqs)